repo/fsrepo/fsrepo.go

```diff
@@ -379,6 +398,61 @@ func (r *FSRepo) openDatastore() error {
 	return nil
 }
 
+var SpecFn = "spec"
```

Comment:
maybe call this `repo_spec`, or `repo_id`. Spec is a very unassuming name that makes me more curious than i should be, calling something `id` generally makes me think I shouldnt touch it